docs/schema-loader.md

```diff
@@ -182,139 +219,184 @@ Create/Delete JDBC schemas
 
 ### Create namespaces and tables
 
-For using a config file (Sample config file can be found [here](https://github.com/scalar-labs/scalardb/blob/master/conf/database.properties)):
+To create a namespace and tables by using a properties file, run the following command, replacing `<VERSION>` with the version of ScalarDB that you're using and `<PATH_TO_PROPERTIES_FILE>` with the path to the properties file. If `--coordinator` is specified, a Coordinator table will be created.
```

Comment:
I guess the `--cordinator` option should be described in the dedicated section because they affect users much. If users want to use transactions with the consensus commit (and maybe almost all ScalarDB users do), they need to create a Coordinator table. However, users would be likely to be unsure what is the Coodinator. So, IMO, it is worth emphasizing what is the Coordinator and who needs to create it here.

Reply:
> To create a namespace and tables

Schema Loader can create [multiple namespaces](https://github.com/scalar-labs/scalardb-samples/blob/main/multi-storage-transaction-sample/schema.json), so I think "To create namespaces and tables" would be better.

Reply:
@choplin I see. How about if we direct users to [this section of the ScalarDB Java API Guide](https://scalardb.scalar-labs.com/docs/latest/api-guide/#specify-operations-for-the-coordinator-table) that briefly describes the Coordinator table?

Or, within this document, do we need to add more details about the Coordinator table that are specific to Schema Loader?

Reply:
> Schema Loader can create [multiple namespaces](https://github.com/scalar-labs/scalardb-samples/blob/main/multi-storage-transaction-sample/schema.json), so I think "To create namespaces and tables" would be better.

@komamitsu Thank you! I've applied your suggestion.

Reply:
> @choplin I see. How about if we direct users to [this section of the ScalarDB Java API Guide](https://scalardb.scalar-labs.com/docs/latest/api-guide/#specify-operations-for-the-coordinator-table) that briefly describes the Coordinator table?

Yes, this would be sufficient. We don't need to duplicate it. Thanks!